### DISCUSSION TITLE
The login interface cannot be displayed

### DISCUSSION THREAD

**Discussion by qq1907** [rustdesk/rustdesk#531] [General]
When the windows desktop is logged in, a black screen is displayed after connecting, and the windows login account and password cannot be entered

![20220213115857](https://user-images.githubusercontent.com/75230744/153738445-e1a182d3-0c50-4548-bd01-665ae348d497.png)
![20220213121402](https://user-images.githubusercontent.com/75230744/153738447-f168f8b3-ca9e-484e-8075-22f1a50d8dad.png)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/96

**Comment by qq1907** [comment]
> #96

After clicking install, open it again, and there will still be a prompt to install on the left. I don't know why it can't be installed normally.

**Comment by rustdesk** [comment]
I will provide another standalone setup file in next release.
